src/common/models/Projection.ts

```diff
@@ -491,6 +514,44 @@ export class Projection {
     );
   }
 
+  private calcCaseDensityCases(
+    actualTimeseries: Array<CANActualsTimeseriesRow | null>,
+  ): Array<number | null> {
+    return actualTimeseries.map(
+      (row: CANActualsTimeseriesRow | null, i: number) => {
+        const weeklyAverage = this.getWeeklyAverageCaseForDay(i);
+        const totalPopulation = this.totalPopulation;
+        if (weeklyAverage) {
+          console.log(
+            'calc cases',
+            weeklyAverage / (totalPopulation * 0.00001),
+          );
+          return weeklyAverage / (totalPopulation * 0.00001);
+        }
+        return null;
+      },
+    );
+  }
+
+  private calcCaseDensityDeaths(
+    actualTimeseries: Array<CANActualsTimeseriesRow | null>,
+  ): Array<number | null> {
+    return actualTimeseries.map(
+      (row: CANActualsTimeseriesRow | null, i: number) => {
+        const weeklyAverage = this.getWeeklyAverageDeathForDay(i);
```

Comment:
Looks like we are missing the .01 factor here.  So we need something like:
```
const estimatedCaseFatalityRatio = .01; // 1% of cases expected to turn into deaths.
const estimatedCases = weeklyAverage / estimatedCaseFatalityRatio;
```
And then use estimatedCases in the calculation below.